### DISCUSSION TITLE
envd dashboard design

### DISCUSSION THREAD

**Discussion by gaocegege** [tensorchord/envd#1055] [Design]
https://www.figma.com/file/KYBe7GYdwTxUt0vRImUICf/envd-dashboard

Still working in progress.

**Comment by shaonianche** [comment]
What do you think about using [kubernetes-dashboard](https://github.com/kubernetes/dashboard)-like styles to build our pages?

**Reply by gaocegege** [reply]
Feel free to add your design to the Figma file. We can discuss it further later.

**Comment by gaocegege** [comment]
What do you mean k8s dashboard style? I used it before but not sure if we are on the same page about its design

**Reply by shaonianche** [reply]
> What do you mean k8s dashboard style? I used it before but not sure if we are on the same page about its design

There are many frameworks in github for building dashboard, look up: https://github.com/search?q=dashboard.
I would like to try to use some kind of framework in the style of the design.
This way the developer can get the design to work at the lowest cost
